pkg/kuttlctl/cmd/test.go

```diff
@@ -145,6 +146,8 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 				options.ArtifactsDir = artifactsDir
 			}
 
+			options.Timeout = timeout
```

Comment:
```suggestion
			if isSet(flags, "timeout") {
						options.Timeout = timeout
			}
```

Reply:
This way the config file only gets overridden if the user specifies `--timeout`, otherwise it would always get set to the CLI's default.